pkg/fs/fs-bucket-listobjects.go

```diff
@@ -20,18 +20,155 @@ import (
 	"fmt"
 	"net/url"
 	"os"
-	"path"
 	"path/filepath"
 	"strings"
+	"time"
 
 	"github.com/minio/minio/pkg/probe"
 )
 
+const (
+	// listObjectsLimit - maximum list objects limit.
+	listObjectsLimit = 1000
+)
+
+// ObjectInfo - object info.
+type ObjectInfo struct {
+	Bucket       string
```

Comment:
As you are introducing this structure here, its better to be fixed in this patch.